portality/formcontext/formcontext.py

```diff
@@ -1191,6 +1213,21 @@ def source2form(self):
         self._set_choices()
         self._expand_descriptions(["publisher", "society_institution", "platform"])
 
+    def pre_validate(self):
```

Comment:
Is this validator exactly the same as the one above at https://github.com/DOAJ/doaj/pull/919/files#diff-a08df963abb007b5bc03439f44aaa7b9R470 ?